site/docs/v1/tech/apis/_webauthn-register-start-request-body.adoc

```diff
@@ -0,0 +1,27 @@
+==== Request Body
+
+[.api]
+[field]#displayName# [type]#[String]# [required]#Required#::
+The primary display name for the new passkey.
+
+[field]#name# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**#::
+The optional name for the passkey. This is meant to disambiguate passkeys with the same [field]#displayName#. When this parameter is omitted, it will be defaulted to a random four character alphabetic string that is unique among a user's passkeys.
+
+[field]#userAgent# [type]#[String]# [optional]#Optional#::
+The user's browser's user agent string. This value is meant for troubleshooting issues with passkeys.
+
+[field]#userId# [type]#[UUID]# [required]#Required#::
+The unique Id of the user registering a new passkey.
+
+[field]#workflow# [type]#[String]# [required]#Required#::
+The WebAuthn workflow this passkey is intended to be used with. Selecting the proper value will provide the best user experience by limiting the registration ceremony to authenticators that support the features ideally suited for each workflow. The selected workflow will change the creation options on the response based on the tenant configuration. The workflow must be enabled at the tenant level and not overridden at the application level. The possible values are:
++
+ * `bootstrap` - The passkey is intended for use with the bootstrap workflow, which allows authenticating using a WebAuthn passkey from a new device.
```

Comment:
These would be great to explore more in a guide.

Reply:
That's the plan 😁